python/ray/serve/constants.py

```diff
@@ -16,9 +16,15 @@
 #: Max concurrency
 ASYNC_CONCURRENCY = int(1e6)
 
+# How often to call the control loop on the controller.
+CONTROL_LOOP_PERIOD_S = 0.1
+
 #: Max time to wait for HTTP proxy in `serve.start()`.
 HTTP_PROXY_TIMEOUT = 60
 
+#: Default try count for allowing failures in replica constructor.
+REPLICA_CONSTRUCTOR_TRY_COUNT = 4
```

Comment:
`REPLICA_CONSTRUCTOR_RETRY_COUNT` would be more intuitive to me (0 == don't retry instead of 1 == don't retry as currently written)

Reply:
that was my initial implementation too as i tried to set default retry count as 3, which means total of 4 tries; I can change it back and modify the retry for loop a bit given it's 0 indexed.